src/containers/flows/MakeABid/MakeABid.tsx

```diff
@@ -83,6 +83,17 @@ const MakeABid: React.FC<MakeABidProps> = ({ domain, onBid }) => {
 	const zAuctionAddress = znsContracts.zAuction.address;
 	const wildContract: ERC20 = znsContracts.wildToken;
 
+	const { isBiddable, isDomainOwner } = useMemo(() => {
+		const isBiddable = Boolean(domainMetadata?.isBiddable);
```

Comment:
Change this logic to `domainMetadata?.isBiddable ?? true`, as `undefined` `isBiddable` should default to `true`

Reply:
Update: talking about this in a different thread - Eric is going to change the metadata retrieval logic to handle this